core/config/config.go

```diff
@@ -0,0 +1,11 @@
+package config
+
+type Config struct {
+	AppVersion        string `envconfig:"APP_VERSION" default:"0.0.0"`
+	Headless          string `envconfig:"HEADLESS" default:"true"`
+	BrowserSourcePort string `envconfig:"BROWSER_SOURCE_PORT" default:"4242"`
```

Comment:
Could make this an integer type? Tiny benefit but would be nice to have that safety i guess. We need to format string anyways when using it